### DISCUSSION TITLE
DIsks with software in dungeon chests

### DISCUSSION THREAD

**Discussion by bastetfurry** [fnuecke/oc2#132] [Ideas]
What the title says, would there be interest if someone would compile a bunch of tools, games and whatnot for the RiscV64 Linux and add them as diskettes to the loot tables of various dungeons?
We already got BywaterBASIC for example and it would have been cool to stumble on a dungeon and find a dusty old floppy with BWBASIC written on it. And for example Rogue in another.

**Comment by fnuecke** [comment]
I'm sure that could be useful for people! I would however like to avoid putting this into the "main" mod, and rather see it in an addon. I'd like to keep the binary blobs in the mod to a minimum, since these are always the most tedious and difficult part to update/maintain.
